lualib/resty/events/worker.lua

```diff
@@ -32,6 +32,7 @@ local decode = codec.decode
 local cjson_encode = cjson.encode
 
 local EVENTS_COUNT_LIMIT = 100
+local EVENTS_POP_LIMIT = 5000
```

Comment:
timerng currently uses a fixed value `5000`. I wonder if this value is too high or too low.

Reply:
For now let's keep consistent with it.

Reply:
Before merge it, I would suggest send this patch to user for test, along with debug.traceback.

Reply:
@chronolaw just checked with @ADD-SP , the default value on timerng side is `2000`. There is a misinfo on the readme.

https://github.com/Kong/lua-resty-timer-ng/blob/aec8b5f109e30cabf2f453222a374dc0bca56157/lib/resty/timerng/constants.lua#L17

Reply:
done.